pages/models/regulations/governors/GovSteamEU/index.md

```diff
@@ -0,0 +1,422 @@
+---
+layout: page
+title: GovSteamEU
+tags: [regulations, governor, controller, synchronous machine, Opensource, IEC, CIMDynamic, RMS, modelica, dynawo]
+author: Martin Franke, Saikrishna Vallabhaneni
+date: 20/06/2024
+last-updated: 20/06/2024
+id: #59
+---
+# GovSteamEU
+
+## Context
+
+The traditional operation of electrical power systems depends on the
```

Comment:
do we have an history f the model ? when did it become a standard? in which of the oldest standard is this model described?

Reply:
We could not find any mentioning of the model in other literature. If anyone knows anything, please share. 
A remark about a 1973 report, explaining electro-hydraulic governors like this one, has been added, though.